### ISSUE TITLE
Wrong title and http status code when rss is disabled

### ISSUE THREAD

**Issue by sshcli** [Kunena/Kunena-Forum#6146]
**Steps to reproduce the behavior:**
1. Configuration->General->Enable RSS Feeds = No
2. Go to example.com/topics/mode-topics?format=feed&type=rss
3. Take a look to the page title and http status code

**Expected behavior**
Page title should be: `404 Not Found`
HTTP Status code is: `404 Not Found`

**Actual result**
Page title is: `Error: 401 Sorry, RSS Feeds have been disabled by administrator.`
HTTP Status code is: `500 internal Server Error`

**System information (please complete the following information)**
Joomla version: 3.9.0
Kunena version: 5.1.6-DEV 2018-11-04
Php version: 7.2.11
Database version: 5.6.41

**Comment by 810** [comment]
I think this is a good message. For fixing the issue.

For the status we need to wait for: https://github.com/joomla/joomla-cms/pull/18966

**Comment by sshcli** [comment]
Yes, we can keep the part `RSS Feeds have been disabled by administrator.` and only remove the part `Error: 401  ` 

it gives the idea to user that something is wrong with the website.

**Comment by sshcli** [comment]
> For the status we need to wait for: joomla/joomla-cms#18966

it was merged 7 hours ago
